### DISCUSSION TITLE
Change of behaviour for the save and exit buttons

### DISCUSSION THREAD

**Discussion by Wladefant** [personalizedrefrigerator/js-draw#40] [Ideas & Feature Requests]
other plugins use the save and close and that seems much more reasonable to me. 
![image](https://github.com/personalizedrefrigerator/js-draw/assets/79611093/f8e2b146-39b1-41ef-bb89-0e8a80e98791)
Something similar @laurent22 requested:
            
- Would it make sense to call the Save/Exit button Save/Cancel? Also please remove the icons from these buttons

- I've just checked a few iOS and Android apps and the Save button is usually on the right, and Cancel on the left so please swap the order

_Originally posted by @laurent22 in https://github.com/laurent22/joplin/issues/7588#issuecomment-1733360562_

About @laurent22 second point, I am not able to see it. Save to the left, close to the right.

**Comment by personalizedrefrigerator** [comment]
I agree with "Save"/"Close" ("Cancel" makes me think of discarding *all* changes).

> I've just checked a few iOS and Android apps and the Save button is usually on the right, and Cancel on the left so please swap the order

Many apps have a "←" back button on the left of the screen. Because `js-draw`'s Close/Exit button does the same thing as pressing the hardware back button in Android, I think it makes sense to put Close in the same place as the back button on other screens.
